packages/fftw3/fftw3.0.8.2/opam

```diff
@@ -14,6 +14,7 @@ build: [
     "env"
     "FFTW3_CFLAGS=-I /usr/local/include -L /usr/local/lib -I /usr/local/include -L /usr/local/lib"
     "dune"
+  "dune-configurator"
```

Comment:
```suggestion
```
This shouldn't be there

Reply:
Actually, I believe it should:

```
$ opam source fft3.0.8.2
[fftw3.0.8.2] downloaded from cache at https://opam.ocaml.org/cache
Successfully extracted to /home/dim/code/fftw3.0.8.2
$ cd fftw3.0.8.2/
$ dune external-lib-deps @install
These are the external library dependencies in the default context:
- bigarray
- dune.configurator
- str
```

Reply:
Ah never-mind, I just realised that it was part of the build instructions...